pyemma/coordinates/estimation/covariance.py

```diff
@@ -27,18 +27,63 @@
 from pyemma._ext.variational.estimators.running_moments import running_covar
 
 
-__all__ = ['EmpiricalCovariance', 'KoopmanEquilibriumCovariance']
+__all__ = ['EmpiricalCovariance']
 
 __author__ = 'paul, nueske'
 
 
-class _CovarEstimator(StreamingEstimator, ProgressReporter, Loggable):
+class LaggedCovariance(StreamingEstimator, ProgressReporter, Loggable):
     def __init__(self, xx=True, xy=False, yy=False, remove_constant_mean=None, remove_data_mean=False, reversible=False,
                  bessel=True, sparse_mode='auto', modify_data=False, lag=0, weights=None, stride=1, skip=0,
                  chunksize=None):
+        """
+        Compute lagged covariances between time series.
+
+        Parameters
+        ----------
+        xx : bool, optional, default=True
+            compute instantaneous correlations over the first part of the data. If lag==0, use all of the data.
+        xy : bool, optional, default=False
+            compute lagged correlations. Does not work with lag==0.
+        xx : bool, optional, default=False
+            compute instantaneous correlations over the second part of the data. Does not work with lag==0.
```

Comment:
Maybe it's more clear if you use ``c00``, ``c0t``, ``ctt`` as in the API function (generally try to make the API functions and corresponding classes look the same where possible)